test/e2e/utils/test_context.go

```diff
@@ -88,27 +88,36 @@ func (kc *KBTestContext) InstallCertManager() error {
 	if _, err := kc.Kubectl.Command("create", "namespace", "cert-manager"); err != nil {
 		return err
 	}
-	_, err := kc.Kubectl.Apply(false, "-f", fmt.Sprintf("https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml", certmanagerVersion), "--validate=false")
+	_, err := kc.Kubectl.Apply(false, "-f",
+		fmt.Sprintf("https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml",
+			certmanagerVersion), "--validate=false")
 	return err
 }
 
 // InstallPrometheusOperManager installs the prometheus manager bundle.
 func (kc *KBTestContext) InstallPrometheusOperManager() error {
-	_, err := kc.Kubectl.Apply(false, "-f", fmt.Sprintf("https://raw.githubusercontent.com/coreos/prometheus-operator/release-%s/bundle.yaml", prometheusOperatorVersion))
+	_, err := kc.Kubectl.Apply(false, "-f",
+		fmt.Sprintf("https://raw.githubusercontent.com/coreos/prometheus-operator/release-%s/bundle.yaml",
+			prometheusOperatorVersion))
 	return err
 }
 
 // UninstallPrometheusOperManager uninstalls the prometheus manager bundle.
 func (kc *KBTestContext) UninstallPrometheusOperManager() {
-	if _, err := kc.Kubectl.Delete(false, "-f", fmt.Sprintf("https://github.com/coreos/prometheus-operator/blob/release-%s/bundle.yaml", prometheusOperatorVersion)); err != nil {
+	if _, err := kc.Kubectl.Delete(false, "-f",
+		fmt.Sprintf("https://github.com/coreos/prometheus-operator/blob/release-%s/bundle.yaml",
+			prometheusOperatorVersion)); err != nil {
 		fmt.Fprintf(GinkgoWriter, "error when running kubectl delete during cleaning up prometheus bundle: %v\n", err)
 	}
 }
 
 // UninstallCertManager uninstalls the cert manager bundle.
 func (kc *KBTestContext) UninstallCertManager() {
-	if _, err := kc.Kubectl.Delete(false, "-f", fmt.Sprintf("https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml", certmanagerVersion)); err != nil {
-		fmt.Fprintf(GinkgoWriter, "warning: error when running kubectl delete during cleaning up cert manager: %v\n", err)
+	if _, err := kc.Kubectl.Delete(false, "-f",
+		fmt.Sprintf("https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml",
+			certmanagerVersion)); err != nil {
```

Comment:
Same as above, two lines

Reply:
Not possible.

Reply:
NOTE: `go fmt ./...` is used here to format the code according to the go good practices and it also is used to check/verify its indentation automatically in the ci.

Reply:
Yeah, but sometimes `go fmt` is not as intelligent as we are in order to decide where should he split the lines. Would this fit in two lines?
```
	if _, err := kc.Kubectl.Delete(false, "-f", fmt.Sprintf("https://github.com/jetstack/cert-manager/" +
		"releases/download/%s/cert-manager.yaml", certmanagerVersion)); err != nil {
```

It is a NIT you can skip if you don't think it helps.

Reply:
The go fmt is the GO standard to format the files. 
This is the rule that needs to be followed. 

If we address your suggestion here we will be splitting the URL which IHMO is not helpful at all.

Reply:
You may be right, splitting the URL may not be a good idea, you can mark resolved.

(What I mean by `go fmt` is that choosing where to split a line can not be optimized as there are too many criteria, so `go fmt` does one thing but you can split in other points that make it clearer)

Reply:
SO, I am closing this one as sorted out feel free to re-open if you see that is needed.